### ISSUE TITLE
Control zoom scale

### ISSUE THREAD

**Issue by matthew9510** [laurenashpole/react-inner-image-zoom#19]
Right now when I zoom in a picture, it zooms too much for my preference.

**Comment by andyseracuse** [comment]
I fixed this by adding a height to the .iiz__zoom-img property to be twice the size of the original image (in my case the original image was set to 550px and I wanted 2.5x zoom, so I set it the height to 1375 px).  The rule looks like this after I was done with it

`.iiz__zoom-img {
  width: auto !important;
  max-width: none !important;
  position: absolute;
  visibility: hidden;
  opacity: 0;
  display: block;
  z-index: 100;
  height: 1375px;
}`

**Comment by ecovirtual** [comment]
A quick fix for changing the scale of the zoomed image would be to just add an extra Height declaration to your styles, with the desired percentage. No need to do much pixel calculation:

`.iiz__zoom-img { height: 300%}`

+1 for adding a 'zoomScale' property to the component.

**Comment by laurenashpole** [comment]
Hey! Sorry I'm a little slow but thanks @andyseracuse and @ecovirtual for offering some temporary solutions. I'll take a look over the weekend to see how complicated adding a scale property would be.

**Comment by thiagomorales** [comment]
Will be great if we could get responsive values for this prop.
I’ve managed it overriding styles like said above.

```
&__zoom-img {
      width: auto;
      height: 250%;
      @media all and (min-width: 992px) {
        height: 400%;
      }
```

**Comment by laurenashpole** [comment]
It turns out this wasn't too complicated. Just released v1.3 with a `zoomScale` prop that gets multiplied by the original width and height of the zoomed image to scale it up or down.

For responsiveness, I'd suggest using something like [react-responsive](https://www.npmjs.com/package/react-responsive) to conditionally set the scale amount since they've already mastered easy media queries and resize listening.
